module/VuFind/src/VuFind/Db/Entity/Shortlinks.php

```diff
@@ -90,4 +91,83 @@ class Shortlinks implements EntityInterface
      * )
      */
     protected $created = 'CURRENT_TIMESTAMP';
+
+    /**
+     * Id getter
+     *
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Shortlinks hash setter
+     *
+     * @param ?string $hash Shortlinks hash.
+     *
+     * @return Shortlinks
+     */
+    public function setHash(?string $hash): Shortlinks
+    {
+        $this->hash = $hash;
+        return $this;
+    }
+
+    /**
+     * Hash getter
+     *
+     * @return string
+     */
+    public function getHash(): string
```

Comment:
If the setter allows `?string`, then shouldn't the return value here also be `?string`? On a related note, the type on the property above could be simplified from `string|null` to `?string` for consistency.